github-connector/Deployment.yaml

```diff
@@ -13,7 +13,8 @@ spec:
     spec:
       containers:
       - name: test1
-        image: karoljaksik/webhook-test:1.0.0
+      
+        image: karoljaksik/webhook-test:1.0.1
```

Comment:
Please consider using kyma-related container storage, you can reach me directly on that.